### PR TITLE
Bitscope

### PR DESCRIPTION
**Pull request by scontini76**
BitScope Application Suite: DSO, Meter, Logic, Chart

### CODE REVIEW

File: Casks/bitscope-chart.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+cask :v1 => 'bitscope-chart' do
+  version '1.1.DC01G'
+  sha256 'e7c2900281cf289e6ae458a04c8435024aa526ef2df8d0b17874e71f9c92c3a8'
+
+  url 'http://bitscope.com/download/files/bitscope-chart_1.1.DC01G.app.tgz'
```
Comment (fanquake) [inline Casks/bitscope-chart.rb]: You should be using #{version} in the url

Comment (scontini76) [inline Casks/bitscope-chart.rb]: Yeah, sorry for the mistake, I'm going to fix it, thank you! :)

**Comment by vitorgalvao** [comment]
Please remove the caveats, and these are good to go.

**Comment by scontini76** [comment]
It's not a problem for me to remove the caveats, but I put them because you are invited to read the Terms and Conditions in order to downloads those applications from the BitScope site (http://my.bitscope.com/download/). Maybe I misunderstand the use of caveats, I'm going to remove them as you asked!

**Comment by vitorgalvao** [comment]
Please remove them, yes.

Caveats should be used only for things that pertain to the installation itself. This kind os agreement acceptance is something we’re still thinking about.

**Comment by scontini76** [comment]
Ok, thank you very much for the explanation!

**Comment by vitorgalvao** [comment]
Thank you for the contribution.

**Pull request merged**